Makefile

```diff
@@ -14,7 +14,7 @@ install:
 example: default
 	protoc -I. -I$(SRCPATH) -I./vendor \
 		--go_out="plugins=grpc:$(SRCPATH)" --gorm_out="$(SRCPATH)" \
-		example/feature_demo/*.proto
+		example/feature_demo/test.proto example/feature_demo/test2.proto
```

Comment:
Wondering because this came up in a discussion earlier, did you have errors with the *.proto notation, or is this just clearer?

Reply:
Not an error, but a little inconsistency across environments. This `example/feature_demo/*.proto` may expand either into `example/feature_demo/test.proto example/feature_demo/test2.proto` or `example/feature_demo/test2.proto example/feature_demo/test.proto` depending on the environment, and thus results in a slightly different generated code(mainly comment headers differ).